src/main/java/core/basesyntax/strategy/impl/BalanceOperationImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProductDao;
+import core.basesyntax.strategy.OperationService;
+
+public class BalanceOperationImpl implements OperationService {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
And so on in all other cases.